core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly

```diff
@@ -87,8 +87,8 @@ THE SOFTWARE.
                                             <j:when test="${useCounter > 0}">
                                                 <j:if test="${token.lastUseDate == null}">
                                                     <!--Should normally never happen without manual XML modification-->
-                                                    <span class="token-use-counter" title="${$TokenNeverUsedTooltip}">
-                                                        ${%TokenNeverUsedValue}
+                                                    <span class="token-use-counter" title="${%TokenNeverUsedTooltip}">
+                                                        ${%TokenNeverUsed}
```

Comment:
`TokenNeverUsedValue` actually doesn't exists and `TokenNeverUsed` is the correct name: https://github.com/jenkinsci/jenkins/blob/jenkins-2.251/core/src/main/resources/jenkins/security/ApiTokenProperty/config.properties#L10

Translations also uses `TokenNeverUsed`.